model_cfg_file: "multi_path.yaml"
model_ckpt_file: "multi_path_checkpoint_epoch_120.pth"
debug_viz: false
use_kitti_bag: true

kitti:
    topic_lidar: "/kitti/velo/pointcloud"
    topic_gps: "/kitti/oxts/gps/fix"
    topic_imu: "/kitti/oxts/imu"
    gps_path: 'kitti_10.csv'
    hmi_path: 'kitti_10_hmi.csv'
